### PR TITLE
Translation update (ca)

### PR DESCRIPTION
**Pull request by dokuwiki-translate**
This pull request contains some translation updates.

### CODE REVIEW

File: inc/lang/ca/lang.php [hunk 1]
```diff
@@ -332,8 +335,12 @@
 $lang['media_perm_upload']     = 'No teniu permisos suficients per a pujar arxius';
 $lang['media_update']          = 'Puja la nova versió';
 $lang['media_restore']         = 'Restaura aquesta versió';
+$lang['media_acl_warning']     = 'Aquesta llista pot no estar completa per restriccions ACL i per llistes ocultes';
 $lang['currentns']             = 'Espai de noms actual';
 $lang['searchresult']          = 'Resultats cerca';
 $lang['plainhtml']             = 'HTML pla';
+$lang['wikimarkup']            = 'Wiki Marcatge';
+$lang['page_nonexist_rev']     = 'Pàgina no existeix a %s. Aixó es conseqüencia d\'haver-la creada a <a href="%s">%s</a>.';
```
Comment (6temes) [inline inc/lang/ca/lang.php]: això
conseqüència
haver-la treat a

File: inc/lang/ca/lang.php [hunk 2]
```diff
@@ -332,8 +335,12 @@
...
+$lang['unable_to_parse_date']  = 'Impossible de esbrinar el paràmetre "%s".';
```
Comment (6temes) [inline inc/lang/ca/lang.php]: Impossible esbrinar...

File: inc/lang/ca/lang.php [hunk 3]
```diff
@@ -283,6 +285,7 @@
 $lang['i_modified']            = 'Per raons de seguretat aquesta seqüència només funciona amb una instal·lació nova i no modificada de Dokuwiki. Hauríeu de tornar a baixar el paquet i/o descomprimir-lo o consultar les <a href="http://dokuwiki.org/install">instruccions d\'instal·lació de Dokuwiki</a> completes';
 $lang['i_funcna']              = 'La funció PHP <code>%s</code> no està disponible. Potser el vostre proveïdor de serveis l\'ha inhabilitada per alguna raó';
 $lang['i_phpver']              = 'La vostra versió de PHP <code>%s</code> és inferior a la requerida <code>%s</code>. Necessiteu actualitzar la vostra instal·lació de PHP.';
+$lang['i_mbfuncoverload']      = 'mbstring.func_overload cal que sigui deshabilitada en php.ini perquè funcioni DokuWiki';
```
Comment (6temes) [inline inc/lang/ca/lang.php]: deshabilitada a l'arxiu php.ini

**Comment by splitbrain** [comment]
@6temes I will merge this. You can send an updated version then through the translation interface

**Comment by 6temes** [comment]
Sorry, I am not user of dokuwiki anymore. I just reviewed because I was assigned by mention-boot, and because I had some spare time.

But it's a pity because this translation is full of grammar mistakes. :/

**Pull request merged**